classes/system/heka/remote_collector/single.yml

```diff
@@ -0,0 +1,14 @@
+classes:
+- service.heka.remote_collector.single
```

Comment:
You need to pull the `service.heka.support` class. See https://github.com/Mirantis/mk-lab-salt-model/pull/53.